### PR TITLE
Issue69 correct urine screening logic

### PR DESCRIPTION
**Pull request by sliver007**

### CODE REVIEW

File: pages/cql/OpioidCDS_DSTU2_REC_10.cql [hunk 1]
```diff
@@ -55,15 +55,14 @@ define "Validate Trigger Event":
   )
 
 define "Inclusion Criteria":
-  AgeInYears() >= 18
+  AgeInYearsAt(Now()) >= 18
     and "Validate Trigger Event"
```
Comment (brynrhodes) [inline pages/cql/OpioidCDS_DSTU2_REC_10.cql]: What prompted the change here? This is using a DateTime (Now()), and so will result in the patient's birth _time_ being considered if it is present. That isn't the intent, so the AgeInYears() is preferred.

File: pages/cql/OpioidCDS_DSTU2_REC_10.cql [hunk 2]
```diff
@@ -55,15 +55,14 @@ define "Validate Trigger Event":
...
     and not "Exclusion Criteria"
     and AnyTrue(Common.ProbableDaysInRange(Common."Active Ambulatory Opioid Rx", 90, 80))
     and
       ("No Urine Screening In Last 12 Months"
-        or "Has Missing Opioids?"
-        or "Has Unprescribed Opioids?"
-        or "Has Illicit Drugs in Screening?")
+        or "Has Evidence of Unprescribed Opioids"
```
Comment (brynrhodes) [inline pages/cql/OpioidCDS_DSTU2_REC_10.cql]: This should just be "Has Evidence of Opioids", we don't have any basis for saying whether they are prescribed or not.

Comment (sliver007) [reply pages/cql/OpioidCDS_DSTU2_REC_10.cql]: I have removed the notion of "prescribed".

**Comment by sliver007** [comment]
Resources Updated

**Pull request merged**